### PR TITLE
Ft auth registration and login

### PR DESCRIPTION
**Pull request by maniplek**
#### What does this PR do?
login and register

#### Description of Task to be completed?
this task aims at registering and login 

#### How should this be manually tested?
Pull this project repository
Create an environment variable with python version 3.10
Run pip install -r requirements.txt

#### Any background context you want to provide?
i can send a token and i login using the token 

#### What are the relevant pivotal tracker stories?
(PR#14)

#### Screenshots (if appropriate)
Screenshot 2022-03-15 072420
Screenshot 2022-03-15 072517
token checking
token
header

### CODE REVIEW

File: apps/authentication_app/admin.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+from django.apps import apps
+# Register your models here.
+
+admin.site.register(User)
+
+app = apps.get_app_config('graphql_auth')
+
+for model_name, model in app.models.items():
+    admin.site.register(model)
+
```
Comment (SamuelIZABAYO) [inline apps/authentication_app/admin.py:12]: Indent well the last lines of code

File: apps/authentication_app/schemas/query/Query.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    def resolve_users(self, info):
+        return User.objects.all()
+
+    def resolve_me(self, info):
+        user = info.context.user
+        if user.is_anonymous:
+            raise Exception('Not logged in!')
+
+        return user
```
Comment (SamuelIZABAYO) [inline apps/authentication_app/schemas/query/Query.py]: Add an empty line at the end of this file

File: scrum/settings.py [hunk 3]
```diff
@@ -136,7 +143,29 @@
...
+    'LOGIN_ALLOWED_FIELDS': ['email', 'username'],
+    # ...
+}
+
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
+
 with open('.env') as f:
-    SECRET_KEY = f.read().strip()
\ No newline at end of file
+    SECRET_KEY = f.read().strip()
```
Comment (SamuelIZABAYO) [inline scrum/settings.py]: remove lines 170-172 and add an empty line at the end of this file

**Review by SamuelIZABAYO — COMMENTED** [review]
Please do consider adding an empty line at the end of each of your files

**Review by kagabof — APPROVED** [review]

**Pull request merged**